### ISSUE TITLE
Add current homework to dashboard

### ISSUE THREAD

**Issue by labrocadabro** [labrocadabro/communitytaught#43]
As with current class, we need current homework on the dashboard.

**Comment by numulaa** [comment]
can you please assign this to me:)

**Comment by labrocadabro** [comment]
Certainly. I think the best way to do this one is to list all incomplete homework items assigned on or before the current class - 1. Meaning if someone is on class 5, this list should show all homework items from classes 1-4 that haven't been marked as complete. This will probably need some changes to the backend to get the necessary information.

**Comment by labrocadabro** [comment]
Just an FYI, the database connection has changed slightly and you'll need to update your .env. See example.env for details.
